cypress/integration/1-getting-started/e2e.spec.js

```diff
@@ -0,0 +1,114 @@
+describe("Open Page", () => {
+    beforeEach(() => {
+      cy.visit('./source/index.html');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group11/cse110-sp21-group11/pullRequest?prid=8548116)
```suggestion
      cy.visit("./source/index.html");
```